scripts/twister

```diff
@@ -1322,5 +1327,6 @@ if __name__ == "__main__":
     try:
         main()
     finally:
-        if os.isatty(1): # stdout is interactive
+        if (os.name != "nt") and os.isatty(1):
```

Comment:
```suggestion
        if os.name != "nt" and os.isatty(1):
```

Reply:
For my point of view it is more readable if there are those parenthesis (`Explicit is better than implicit.` from PEP 20 - https://www.python.org/dev/peps/pep-0020/). But if for some reason it is important to you, please explain and I'll change it :)